src/main/java/uk/gov/hmcts/reform/professionalapi/controller/request/DxAddressCreationRequestValidator.java

```diff
@@ -7,9 +7,10 @@
 
 @Service
 @Slf4j
-public class DxAddressCreationRequestValidator implements OrganisationRequestValidator {
+public class DxAddressCreationRequestValidator implements IRequestValidator {

```

Comment:
Interface name should be proper
Class implementing interface should have Impl in name ending